index.ts

```diff
@@ -202,7 +201,7 @@ export default function sveltePlugin(options?: esbuildSvelteOptions): Plugin {
                 dependencyModifcationTimes.set(args.path, statSync(args.path).mtime); // add the target file
 
                 let compilerOptions = {
-                    css: (svelteVersion < 3 ? false : "external") as boolean | "external",
+                    css: (svelteVersion < 3 ? undefined : "external") as CompileOptions["css"],
```

Comment:
After looking at this a bit more, I'm fairly sure this logic is broken for everything between Svelte 3.0.0 and 3.53.0 (when support for "external" was added). Fun.